rpm/quota-4.04-warnquota-configuration-tunes.patch

```diff
@@ -0,0 +1,76 @@
+From 67a0dbf6434552e720b0d311597553b3a76f779e Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
+Date: Tue, 23 Aug 2011 13:45:15 +0200
+Subject: [PATCH] warnquota configuration tunes
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Rest of changes (378a64006bb1e818e84a1c77808563b802b028fa) not
+accepted by upstream (we had root@... addresses and more enterprise
+wordings usually there).
+
+Signed-off-by: Petr Písař <ppisar@redhat.com>
+---
+ warnquota.c    |  2 +-
+ warnquota.conf | 17 ++++++++---------
+ 2 files changed, 9 insertions(+), 10 deletions(-)
+
+diff --git a/warnquota.c b/warnquota.c
+index 0d911e4..2c8e084 100644
+--- a/warnquota.c
```

Comment:
This all seems to be to do with sending warning email, which doesn't make sense on a mobile device.

Reply:
Yeah, I guess you are right. I removed also warnquota subpackage. If someone thinks they need it, it's not that difficult to bring back.